psi_cardinality/c/psi_cardinality_client_c.cpp

```diff
@@ -0,0 +1,73 @@
+#include "psi_cardinality_client_c.h"
+
+#include "internal_utils.h"
+#include "psi_cardinality_client.h"
+
+using namespace psi_cardinality;
```

Comment:
[Avoid `using namespace`](https://google.github.io/styleguide/cppguide.html#Namespaces).
What you could do is import the classes you want available into an anonymous namespace, i.e.,
```C++
namespace {
  using psi_cardinality::PSICardinalityClient;
}
```